core/lib/presentation/resources/image_paths.dart

```diff
@@ -12,7 +12,7 @@ class ImagePaths {
   String get icDownload => _getImagePath('ic_download.svg');
   String get icMore => _getImagePath('ic_more.svg');
   String get icPhotoLibrary => _getImagePath('ic_photo_library.svg');
-  String get icEmptyEmail => _getImagePath('ic_empty_email.svg');
+  String get icEmptyEmail => _getImagePath('ic_empty_email.png');
```

Comment:
why we need  to change it to PNG?

Reply:
PNG increase app too much, can we get support from design team for it?

Reply:
SVG image has an error when exported